src/pages/Home.js

```diff
@@ -138,14 +143,15 @@ const EditButtonWrapper = styled.div`
 `;
 
 const Headline = styled.h1`
-  display: flex;
   align-items: center;
+  color: var(--secondary-dark);
+  display: flex;
```

Comment:
flex works here, but you don't really need it here I think. Could be replaced by text-align: center